README.md

```diff
@@ -1,2 +1,86 @@
-# forum
-openedx forum app
\ No newline at end of file
+# Forum App Installation Guide for Tutor Environment
+
+## Description
+
+The Edly Forum app is designed to replace the [CS Comments Service](https://github.com/openedx/cs_comments_service), providing a more robust and flexible forum solution for Open edX instances. This new forum is built in Python and uses MySQL, which enhances performance and maintainability compared to the previous Ruby and MongoDB implementation. The motivation for this rewrite stems from a need for a more streamlined and integrated discussion platform within Open edX, as outlined in the [Axim Funded Contribution Proposal](https://discuss.openedx.org/t/axim-funded-contribution-proposal-forum-rewrite-from-ruby-mongodb-to-python-mysql/12788).
+
+## Prerequisites
+
+Before you begin, ensure you have the following installed:
+
+- A working Open edX instance using the Tutor platform.
+
+## Installation Steps
+
+Follow the steps below to install the Forum app in your Tutor environment:
+
+### 1. Add a Forum App to the openedX dependencies
+
+Open your Tutor configuration file, typically located at `~/.tutor/config.yml`. Add the following package in the `OPENEDX_EXTRA_PIP_REQUIREMENTS`. You can also access the config file via `vim "$(tutor config printroot)/config.yml"` . We are installing the package from Git because the package isn’t yet available on PyPi.
+
+```bash
+OPENEDX_EXTRA_PIP_REQUIREMENTS = [
```

Comment:
It's better to not edit the file manually, and to run instead: 

    tutor config save --append 'OPENEDX_EXTRA_PIP_REQUIREMENTS=git+https://github.com/edly-io/forum.git@master'